lightwood/mixers/helpers/transformer.py

```diff
@@ -7,17 +7,21 @@ def __init__(self, input_features, output_features):
         self.output_features = output_features
 
         self.feature_len_map = {}
+        self.input_idxs = {}
```

Comment:
why is input_idxs required now?

Reply:
So that the ARNet knows which indexes of the input belong to the (normalized) previous values of the time series